utility/make_private_namespace.py

```diff
@@ -0,0 +1,45 @@
+""" Create a namespace """
+import os
+import subprocess
+import shlex
+from utility import templating
+from ocs.utils import create_oc_resource
+
+
+def make_private_namespace(name, region='us-east-2'):
+    """
+    Create a namespace.
+
+    Args:
+        name (str): namespace to be created
```

Comment:
please use capital letters in the beginning of description

Reply:
Done